File: src/com/bulletphysics/collision/shapes/ConeShapeZ.java
package com.bulletphysics.collision.shapes;

/**
 * ConeShape implements a cone shape, around the Z axis.
 *
 * @author jezek2
 */
public class ConeShapeZ extends ConeShape {

	public ConeShapeZ(double radius, double height) {
		super(radius, height);
		setConeUpIndex(2);
	}

}

File: src/com/bulletphysics/collision/shapes/ConeShapeZ.kt
package com.bulletphysics.collision.shapes

/**
 * ConeShape implements a cone shape, around the Z axis.
 *
 * @author jezek2
 */
class ConeShapeZ(radius: Double, height: Double) : ConeShape(radius, height) {
    init {
        coneUpIndex = 2
    }
}
